testing/tests/install-ignore-dirty

```diff
@@ -0,0 +1,16 @@
+# Test invalid package names and paths
+# @TEST-EXEC: bash %INPUT 2>output
+# @TEST-EXEC: btest-diff output
+
+CONFIG=$(pwd)/config
+
+mkdir ./packages/dirtyrepo
+( cd ./packages/dirtyrepo && \
+  git init . && \
+  touch README && \
+  cp ${PACKAGES}/foo/zkg.meta zkg.meta && \
+  git add README zkg.meta && \
+  git commit -m "Initial commit")
+echo README > ./packages/dirtyrepo/README
+
+zkg --config=$CONFIG install --force --ignore-dirty-git ./packages/dirtyrepo
```

Comment:
In addition to `install`, let's add a similar btest for `test` and `bundle`.

Reply:
Renamed the test to `ignore-dirty` and added tests for `test` and `bundle`.